Command/GenerateAdminCommand.php

```diff
@@ -62,6 +62,14 @@ public function configure()
     /**
      * {@inheritDoc}
      */
+    public function isEnabled()
+    {
+        return class_exists('Sensio\\Bundle\\GeneratorBundle\\SensioGeneratorBundle');
```

Comment:
The command can work if the bundle isn't loaded (it only need some classes from the bundle).